ManagementPacks/Community.DataOnDemand/Modules/GetWindowsUpdateHistoryProbe.mpx

```diff
@@ -0,0 +1,89 @@
+        <Configuration>
+          <xsd:element name="ExcludedKB" type="xsd:string" xmlns:xsd="http://www.w3.org/2001/XMLSchema" />
+          <xsd:element name="Format" type="xsd:string" xmlns:xsd="http://www.w3.org/2001/XMLSchema" />
+          <xsd:element name="TimeoutSeconds" type="xsd:int" xmlns:xsd="http://www.w3.org/2001/XMLSchema" />
+          <xsd:element name="LastHours" type="xsd:string" xmlns:xsd="http://www.w3.org/2001/XMLSchema" />
+          <xsd:element name="ShowTop" type="xsd:string" xmlns:xsd="http://www.w3.org/2001/XMLSchema" />
+        </Configuration>
+        <OverrideableParameters>
+          <OverrideableParameter ID="ExcludedKB" Selector="$Config/ExcludedKB$" ParameterType="string" />
+          <OverrideableParameter ID="Format" Selector="$Config/Format$" ParameterType="string" />
+          <OverrideableParameter ID="ShowTop" Selector="$Config/ShowTop$" ParameterType="string" />
+          <OverrideableParameter ID="LastHours" Selector="$Config/LastHours$" ParameterType="string" />
+          <OverrideableParameter ID="TimeoutSeconds" Selector="$Config/TimeoutSeconds$" ParameterType="int" />
+        </OverrideableParameters>
+        <ModuleImplementation>
+          <Composite>
+            <MemberModules>
+              <ProbeAction ID="Probe" TypeID="Windows!Microsoft.Windows.PowerShellTriggerOnlyProbe">
+                <ScriptName>Get-WindowsUpdateHistory.ps1</ScriptName>
+                <ScriptBody>$IncludeFileContent/Scripts/Get-WindowsUpdateHistory.ps1$</ScriptBody>
+                <Parameters>
+                  <Parameter>
+                    <Name>ExcludedKB</Name>
+                    <Value>$Config/ExcludedKB$</Value>
+                  </Parameter>
+                  <Parameter>
+                    <Name>Format</Name>
+                    <Value>$Config/Format$</Value>
+                  </Parameter>
+                  <Parameter>
+                    <Name>ShowTop</Name>
+                    <Value>$Config/ShowTop$</Value>
+                  </Parameter>
+                  <Parameter>
+                    <Name>LastHours</Name>
+                    <Value>$Config/LastHours$</Value>
+                  </Parameter>
+                </Parameters>
+                <TimeoutSeconds>$Config/TimeoutSeconds$</TimeoutSeconds>
+              </ProbeAction>
+            </MemberModules>
+            <Composition>
+              <Node ID="Probe" />
+            </Composition>
+          </Composite>
+        </ModuleImplementation>
+        <OutputType>Windows!Microsoft.Windows.SerializedObjectData</OutputType>
+        <TriggerOnly>true</TriggerOnly>
+      </ProbeActionModuleType>
+    </ModuleTypes>
+  </TypeDefinitions>
+
+  <LanguagePacks>
+    <LanguagePack ID="ENU" IsDefault="true">
+      <DisplayStrings>
+        <DisplayString ElementID="Community.DataOnDemand.Probe.GetWindowsUpdateHistory">
+          <Name>Data On Demand Windows windows update history probe action</Name>
```

Comment:
This name is a little weird, does it need to conform to a particular convention?

Reply:
It came from that, but now you point it out it of course makes perfect sense to remove the second instance.